### PR TITLE
run-a-node: Add Damask Upgrade to Upgrade Log

### PR DESCRIPTION
**Pull request by tjanez**

### CODE REVIEW

File: docs/general/run-a-node/upgrade-log.md [hunk 1]
```diff
@@ -6,6 +6,179 @@ description: >-
...
+:::info
+
+At this time only entities which have active validator nodes scheduled in the
+validator set are eligible to vote for governance proposals.
+
+:::
+
+:::danger
+
+At least **75%** of the **voting power** needs to cast vote on the upgrade
```
Comment (peterjgilbert) [inline docs/general/run-a-node/upgrade-log.md]:
```suggestion
At least **75%** of the total **voting power** of the validator set needs to cast a vote on the upgrade
```

File: docs/general/run-a-node/upgrade-log.md [hunk 2]
```diff
@@ -6,6 +6,179 @@ description: >-
...
+proposal for the result to be valid.
+
+At least **90%** of the votes need to be **yes** votes for a proposal to be
```
Comment (peterjgilbert) [inline docs/general/run-a-node/upgrade-log.md]:
```suggestion
At least **90%** of the vote needs to be **yes** votes for a proposal to be
```

File: docs/general/run-a-node/upgrade-log.md [hunk 3]
```diff
@@ -6,6 +6,179 @@ description: >-
...
+transaction:
+
+```bash
+oasis-node governance gen_cast_vote \
+  "${TX_FLAGS[@]}" \
+  --vote.proposal.id 2 \
+  --vote yes \
+  --transaction.file tx_cast_vote.json \
+  --transaction.nonce $NONCE \
+  --transaction.fee.gas 2000 \
```
Comment (peterjgilbert) [inline docs/general/run-a-node/upgrade-log.md:134]: Is this enough gas? Has someone confirmed?

Comment (tjanez) [reply docs/general/run-a-node/upgrade-log.md:134]: I haven't tried but we had the same instructions for the last on-chain voting: https://docs.oasis.dev/general/run-a-node/upgrade-log#instructions---voting

File: docs/general/run-a-node/upgrade-log.md (hunk 2 above)
Comment (tjanez) [reply docs/general/run-a-node/upgrade-log.md]: Thanks. Updated.

File: docs/general/run-a-node/upgrade-log.md (hunk 1 above)
Comment (tjanez) [reply docs/general/run-a-node/upgrade-log.md]: Thanks. Updated.

**Review by peterjgilbert — APPROVED** [review]

**Pull request merged**